src/inn.js

```diff
@@ -3,9 +3,8 @@ class Inn {
     this.items = items;
   }
   updateItems() {
-    for (let i = 0; i < this.items.length; i++) {
-        this.items[i].updateItems();
-    }
-
+     this.items.forEach((item) => {
```

Comment:
Do you need the parentheses around the parameter name in JavaScript? Languages I've used don't require them if there's a single parameter, not so sure about JavaScript. 

Also your indentation in this function is off by 1 character, and within the lambda as well. You seem to have mixed indentation styles in the project as a whole: some (most?) files use 4-space indents; this uses 2- or 3-space indents. Be careful of your code style as a whole; most companies have a house style which you will be expected to conform to, but not all of them will have a linter or similar to enforce it. We use IDE settings to enforce our coding style so the developers don't have to think about it at all.